### ISSUE TITLE
Doesn't accept Icon Components

### ISSUE THREAD

**Issue by salientknight** [jpuri/react-draft-wysiwyg#872]
I am trying to replace the icons with material-ui icons but they are not rendered. 
`import FormatBold from "@material-ui/icons/FormatBold";` 
later used as 
` bold: { icon: FormatBold, className: undefined },`

And I get a blank button.

**Comment by St1myL** [comment]
Same problem, it tries to insert material icon as img src:

![image](https://user-images.githubusercontent.com/26415925/73593714-b7e0a400-450f-11ea-8e58-9c946ac495dc.png)

**Comment by vhbert** [comment]
Same issue here, was wondering if this was getting looked at?

**Comment by gamegee** [comment]
Same issue here

**Comment by JacobTheHeroDev** [comment]
Yup.  As per [the source code](https://github.com/jpuri/react-draft-wysiwyg/blob/f59ee8419cdbd45aab3bdfdf1995f112b09bbb6a/src/controls/Inline/Component/index.js#L45), it's just passing the `option.icon` value directly to the `image.src` attribute.  So we're limited to a path, URL, or base64-encoded image.

@jpuri it would be nice to be able to pass in a component for the icon itself, or a component for the option.  As I'm reading the docs and playing with the toolbar options myself, it appears the lower-level options (ie, bold) cannot be overridden.  So the only way I can modify the "bold" icon is to either base64-encode my entire icon library (which are already all React components), or pass in a custom component for the higher-level options (e.g. "inline"), and have it render custom components for each lower-level option, overriding the rendered component there.  This is a lot more work than I originally anticipated, unfortunately.

**Comment by jacob-israel-turner** [comment]
Yup. As per the source code, it's just passing the option.icon value directly to the image.src attribute. So we're limited to a path, URL, or base64-encoded image.

@jpuri it would be nice to be able to pass in a component for the icon itself, or a component for the option. As I'm reading the docs and playing with the toolbar options myself, it appears the lower-level options (ie, bold) cannot be overridden. So the only way I can modify the "bold" icon is to either base64-encode my entire icon library (which are already all React components), or pass in a custom component for the higher-level options (e.g. "inline"), and have it render custom components for each lower-level option, overriding the rendered component there. This is a lot more work than I originally anticipated, unfortunately.
